### PR TITLE
[IOTDB-2743] ttl cannot be set if storage group is not exist

### PR DESCRIPTION
**Pull request by cigarl**
We should tell the user that they are using the wrong storage group,then let them troubleshoot the problem.

### CODE REVIEW

**Comment by HTHou** [comment]
Hi, create a jira issue and add some test about it?

**Comment by cigarl** [comment]
> Hi, create a jira issue and add some test about it?

done. 
PTAL~

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java [hunk 1]
```diff
@@ -1496,14 +1496,20 @@ private void operateUnloadFile(OperateFilePlan plan) throws QueryProcessExceptio
...
           IoTDB.metaManager.getMatchedStorageGroups(plan.getStorageGroup(), plan.isPrefixMatch());
+      if (storageGroupPaths.isEmpty()) {
+        throw new StorageGroupNotSetException(plan.getStorageGroup().getFullPath());
+      }
       for (PartialPath storagePath : storageGroupPaths) {
         IoTDB.metaManager.setTTL(storagePath, plan.getDataTTL());
         StorageEngine.getInstance().setTTL(storagePath, plan.getDataTTL());
       }
+    } catch (StorageGroupNotSetException e) {
+      throw e;
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1512]: This don't need to catch here, just throw it.

Comment (cigarl) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1512]: `StorageGroupNotSetException `extends from `MetadataException`, and `MManager `throws `MetadataException`.
The `StorageGroupNotSetException `here will be caught by `MetadataException`. So individually catch and throw it.

Comment (wangchao316) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1512]: ok

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by wangchao316 — CHANGES_REQUESTED** [review]
test failed, you could see..

**Comment by cigarl** [comment]
@HTHou @wangchao316 
 Sorry, i just got back from other threads.I reviewed the issue and there are two major changes:
 a. The related unit tests bypass the `planExecutor` and validate the `LocalSchemaProcessor` directly.
 b. The test for TTL in `IoTDBTtlIT` catches the exception as a `TIMESERIES_NOT_EXIST`, which is obviously not correct.
PTAL~

**Pull request closed without merging**